### ISSUE TITLE
ui-select : clicking on the arrow does not open the dropdown

### ISSUE THREAD

**Issue by GMouron** [LuccaSA/lucca-ui#101]

**Comment by lucienbertin** [comment]
I think it's because on a `ui-select` with bootstrap styling, the arrow icon has a ng-click event

```html
<i class="caret pull-right" ng-click="$select.toggle($event)"></i>
```

but with lucca-ui, the arrow comes from stylin on `::before` of the `ui-select-container` and doesn't have a `ng-click` so it may come from that

**Comment by bpaugam** [comment]
Fixed https://github.com/LuccaSA/lucca-ui/commit/1afa796a15b793e5d6763a9c60e46d665b624525

**Comment by lucienbertin** [comment]
dont close issues until fixed in an official release thx

**Comment by lucienbertin** [comment]
should be fixed in next release (release [2.0.6](https://github.com/LuccaSA/lucca-ui/releases/tag/2.0.6))
